server/views/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,5 @@
+.car-notification-banner {
+  .govuk-notification-banner__heading {
```

Comment:
Why the `.govuk-notification-banner__heading` part? Is it not enough to apply it to `.car-notification-banner`?

Reply:
The notification banner is already 100% width, it's the heading that's got a limited width so I need to target that specifically to override it.